topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -0,0 +1,316 @@
+> 1. **JBrowse genome browser** {% icon tool %}:
+>    - *"Reference genome to display"*: `Use a genome from history`
+>        - *"Select a reference genome"*: `polished-assembly.fasta`
+>    - *"Produce Standalone Instance"*: `Yes`
+>    - *"Genetic Code"*: `11. The Bacterial, Archaeal and Plant Plastid Code`
+>    - *"JBrowse-in-Galaxy Action"*: `New JBrowse instance`
+>    - *"Insert Track Group"*
+>        - *"Insert Annotation Track"*
+>            - *"Track Type"*: `BAM pileups`
+>            - *"BAM track data"*: `nanopore-tiny.bam`
+>            - *"Autogenerate SNP track"*: `No`
+>            - *Leave the other track features as default*
+>        - *"Insert Annotation Track"*.
+>            - *"Track Type"*: `BAM pileups`
+>            - *"BAM track data"*: `illumina-tiny.bam`
+>            - *"Autogenerate SNP track"*: `No`
+>            - *Leave the other track features as default*
+{: .hands_on}
+
+There is one output file: re-name: `assembly-and-reads`
+* Click on the eye icon to view. (For more room, collapse Galaxy side menus with corner < > signs).
+*  Make sure the bam files are ticked in the left hand panel.
+* Choose a contig in the drop down menu. Zoom in and out with + and - buttons.
+
+Here is an embedded snippet showing JBrowse and the mapped reads:
+
+{% include snippets/jbrowse.html datadir="data" %}
+
+
+> ### {% icon question %} Questions
+>
+> 1. What are the differences between the nanopore and the illumina reads?
+> 2. What are some reasons that the read coverage may vary across the reference genome?
+>
+> > ### {% icon solution %} Solutions
+> > 1. Nanopore reads are longer and have a higher error rate.  
+> > 2. There may be lots of reasons for varying read coverage. Some possibilities: In areas of high read coverage: this region may be a collapsed repeat. In areas of low or no coverage: this region may be difficult to sequence; or, this region may be a misassembly.
+> {: .solution}
+{: .question}
+
+# Annotate the assembly
+
+We can now annotate our assembled genome with information about genomic features.
+* A chloroplast genome annotation tool is not yet available in Galaxy, although for an approximation we could use the tool for bacterial genome annotation, Prokka.
+* However here we will use a web-based tool designed for chloroplast genomes to annotate our assembly (or, you can upload an example annotation).
+
+Follow these steps to annotate your assembly, or upload an example annotation file from this [Zenodo link](http://doi.org/10.5281/zenodo.4267171) to your current Galaxy history, and skip to the next step.
+
+> ### {% icon hands_on %} Hands-on: Annotate with GeSeq
+> * Download `polished.fasta` to your computer (click on the file in your history; then click on the disk icon).
+> * In a new broswer tab, go to [Chlorobox](https://chlorobox.mpimp-golm.mpg.de/geseq.html) where we will use the [GeSeq tool](https://academic.oup.com/nar/article/45/W1/W6/3806659) to annotate our sequence.
```

Comment:
This should use a citation, probably, rather than a link to the article.

Reply:
perfect, thank you!